dll/shellext/deskadp/deskadp.c

```diff
@@ -308,28 +308,48 @@ InitDisplayAdapterDialog(PDESKDISPLAYADAPTER This)
         LocalFree((HLOCAL)lpAdapterName);
     }
 
+    WCHAR lpNA[64];
+     if (!LoadString(hInstance,
+                        IDS_NOTAVAIL,
+                        lpNA,
+                        sizeof(lpNA) / sizeof(lpNA[0])));
+
     if (This->DeskExtInterface != NULL)
     {
+        DPRINT1("Adapter found : CHIP : %s DAC : %s MEM: %s ADAPT %s BIOS : %s\n",
```

Comment:
Do we really need to show this in the debug log?

Reply:
ok